dev-python/haiku-pyapi/haiku_pyapi-0.3.recipe

```diff
@@ -14,8 +14,11 @@ COPYRIGHT="2023-2024 Elozor Bruce
 LICENSE="MIT"
 REVISION="1"
 SOURCE_URI="https://github.com/coolcoder613eb/Haiku-PyAPI/archive/refs/tags/v$portVersion.tar.gz"
-CHECKSUM_SHA256="a534151ba988a406da5e278db5af127fca257c6b260374be054a385335b779d2"
+CHECKSUM_SHA256="3c78f37fc64bd9fbdfedf033f57136dc4f8f05f6872bb60d21c182c1ceddf330"
 SOURCE_DIR="Haiku-PyAPI-$portVersion"
+srcGitRev="d8b1541168e58444597f25e85e78356de9b67d34"
+SOURCE_URI_2="https://github.com/pybind/pybind11/archive/$srcGitRev.zip"
```

Comment:
There is already a pybind11 package, please update it if needed then add a build dependency on it.

Reply:
Hmm, so @OscarL suggested we use a submodule:

https://github.com/coolcoder613eb/Haiku-PyAPI/pull/7

And on IRC, OscarL has stated "(I really don't want to pack pybind11 for every possible Python version on Haiku... when you can just `pip3 install` it without much issues, or just use it as a submodule."

Also, we use the smart_holder branch of pybind11 since it provides a feature that we need. So, we would probably need to make a separate variant of the pybind11 package if we wanted to go that route.

From the PR, I see OscarL gave a suggestion for how to avoid using a second `SOURCE_URI`:

> If this was adopted, then releases would need to be manually supplemented with a tarball that actually includes the content of the submodule(s) (as the default git-archive used by Github won't include submodules's code).
> 
> Just in case, [here's a tool (git-archive-all)](https://github.com/Kentzo/git-archive-all) that can help with that.

Reply:
Honestly just make a release package at Bemini with everything it needs and make it a package,  no need for Haikuports.

Reply:
Well, we would at least like to publish the new version of Haiku-PyAPI...

Reply:
Please only one version, specify the pybind11 branch used for SOURCE_URI_2. That should be OK.